app/views/tutors/find_students.html.erb

```diff
@@ -81,7 +81,7 @@
                                                         <p> Course: <b> <%= course_obj.name %> </b>
                                                         <p> Student: <b> <%= tutee.first_name + ' ' + tutee.last_name%> </b></p>
                                                         <p> Subject: <b> <%= req.subject%> </b></p>
-                                                        <p class="last"> <button class="btn btn-primary" onclick="showEmailForm(['<%=Tutee.find(req.tutee_id).email%>', '<%=req.tutee_id%>', '<%=req.id %>' ])">Select Student</button></p>         
+                                                        <p class="last"> <button class="btn btn-primary" onclick="showEmailForm(['<%=Tutee.find(req.tutee_id).email%>', '<%=req.tutee_id%>', '<%=req.id %>', '<%=req.meeting_length %>' ])">Select Student</button></p>
```

Comment:
Interesting how the type for meeting_length gets converted into an integer when the source is a string. Might have to reorganize the schemas later. Also the pre-existing JavaScript form is doing some pretty hacky stuff. Still works though, so LGTM